src/app/components/accordion/accordion.component.ts

```diff
@@ -1,10 +1,11 @@
-import { CUSTOM_ELEMENTS_SCHEMA, Component, Output, EventEmitter } from '@angular/core';
+import {  Component, Output, EventEmitter } from '@angular/core';
+import { TegelModule } from '@scania/tegel-angular';
 
 @Component({
   selector: 'app-accordion',
   templateUrl: './accordion.component.html',
   standalone: true,
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
We wouldn't have to import it everywhere if it wasn't for standalone: true, right?

Reply:
yes

Reply:
That is correct!